lib/6to5/transformation/transformer.js

```diff
@@ -4,6 +4,25 @@ var traverse = require("../traverse");
 var t        = require("../types");
 var _        = require("lodash");
 
+function noop() { }
```

Comment:
so interestingly, its worth considering not using a sharing function (between many libraries), as it becomes a sharedFunction internally, and can easily become unstable to unrelated usage.